addon/styles/_frost-sidebar.scss

```diff
@@ -1,62 +1,52 @@
-.frost-sidebar .container {
-  width: 100%;
-  height: 100%;
-}
-
 .frost-sidebar {
   position: absolute;
-  left: 0;
+  left: 0px;
+
+  .liquid-container > .liquid-child {
+    width: auto;
+    height: auto;
+  }
+
+  .open-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
```

Comment:
Is this the content container or the component's personal layout?  If it's the content container I don't think we should specify anything, that should be left to the content